MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.NumericUpDown.xaml

```diff
@@ -0,0 +1,58 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:po="http://schemas.microsoft.com/winfx/2006/xaml/presentation/options"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:converters="clr-namespace:MaterialDesignThemes.Wpf.Converters"
+                    xmlns:local="clr-namespace:MaterialDesignThemes.Wpf"
+                    xmlns:system="clr-namespace:System;assembly=mscorlib"
+                    xmlns:transitions="clr-namespace:MaterialDesignThemes.Wpf.Transitions"
+                    xmlns:wpf="clr-namespace:MaterialDesignThemes.Wpf">
+  <ResourceDictionary.MergedDictionaries>
+    <!--
+      we only bring in the dictionaries for controls which were "invented"
+      as part of Material Design in XAML Toolkit. Other themes, for existing controls
+      must be selected manually by the user, so that they override default styles.
+      The easiest way to do this is include:
+      
+      MaterialDesignTheme.Defaults.xaml
+      in your App.xaml
+    -->
+  </ResourceDictionary.MergedDictionaries>
+  <!-- set up default styles for our custom Material Design in XAML Toolkit controls -->
+  <Style TargetType="{x:Type wpf:NumericUpDown}">
+    <Setter Property="MinWidth" Value="100"/>
+    <Setter Property="Padding" Value="4"/>
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="{x:Type wpf:NumericUpDown}">
+          <Border Background="{TemplateBinding Background}"
+                  BorderBrush="{TemplateBinding BorderBrush}"
+                  BorderThickness="{TemplateBinding BorderThickness}"
+                  Padding="{TemplateBinding Padding}"
+                  CornerRadius="10"
+                  ClipToBounds="True">
+            <DockPanel>
+
+              <RepeatButton x:Name="PART_IncreaseButton"
+                            Content="{local:PackIcon Kind=Plus}"
+                            DockPanel.Dock="Right"
+                            Padding="8,4"
+                            Command="{TemplateBinding IncreaseCommand}"/>
+              <RepeatButton x:Name="PART_DecreaseButton"
+                            Content="{local:PackIcon Kind=Minus}"
+                            DockPanel.Dock="Left"
+                            Padding="8,4"
+                            Command="{TemplateBinding DecreaseCommand}"/>
+              
+              <TextBox x:Name="PART_TextBoxField"
+                       Margin="4 0"
+                       VerticalAlignment="Center"
+                       HorizontalContentAlignment="Center" />
```

Comment:
We probably want to consider forwarding the HorizontalContentAlignment from the parent here, and default the value to centered in the Style setters.
```suggestion
                       HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}" />
```